spec.html

```diff
@@ -10669,7 +10669,7 @@ <h1>Function Environment Records</h1>
                 GetSuperBase()
               </td>
               <td>
-                Return the object that is the base for `super` property accesses bound in this Environment Record. The value *undefined* indicates that `super` property accesses will produce runtime errors.
+                Return the object that is the base for |SuperProperty| accesses bound in this Environment Record. The value *undefined* indicates that such accesses will produce runtime errors.
```

Comment:
I'm fine with the `such` change, but I do prefer the original `` `super` property`` phrasing.

Reply:
Restored, but it still reads a little awkwardly to me. What would @tc39/ecma262-editors think about adding parenthetical clarification?
```suggestion
                Return the object that is the base for `super` property accesses bound in this Environment Record (i.e., instances of |SuperProperty|). The value *undefined* indicates that evaluation of such accesses will produce runtime errors.
```

Reply:
The parenthetical doesn't really seem helpful to me personally.